Documentation/sample-metrics.md

```diff
@@ -13,7 +13,7 @@ return the full set of metrics that the Telemeter client captures:
+max(federate_samples - federate_filtered_samples)
```

Comment:
the metrics only show the total number of the metrics, don't know what metrics are collected by the Telemeter service
```
# token=`oc sa get-token prometheus-k8s -n openshift-monitoring`
# oc -n openshift-monitoring exec -c prometheus prometheus-k8s-0 -- curl -k -H "Authorization: Bearer $token" 'https://thanos-querier.openshift-monitoring.svc:9091/api/v1/query?' --data-urlencode 'query=max(federate_samples - federate_filtered_samples)' | jq
{
  "status": "success",
  "data": {
    "resultType": "vector",
    "result": [
      {
        "metric": {},
        "value": [
          1653271388.85,
          "520"
        ]
      }
    ]
  }
}
```
https://github.com/openshift/cluster-monitoring-operator/blob/39a4e00c94de39d436f2a2a605e06c892a2fe809/Documentation/sample-metrics.md

Reply:
Yes, I originally updated this wrongfully.